wagtail_localize/tests/utils.py

```diff
@@ -26,14 +25,8 @@ def make_test_page(parent, cls=None, **kwargs):
 
 
 def get_snippet_list_url_from_args(app_label, model_name):
-    if WAGTAIL_VERSION >= (4, 0):
-        return reverse(f"wagtailsnippets_{app_label}_{model_name}:list")
-
-    return reverse("wagtailsnippets:list", args=[app_label, model_name])
+    return reverse(f"wagtailsnippets_{app_label}_{model_name}:list")
 
 
 def get_snippet_add_url_from_args(app_label, model_name):
-    if WAGTAIL_VERSION >= (4, 0):
-        return reverse(f"wagtailsnippets_{app_label}_{model_name}:add")
-
-    return reverse("wagtailsnippets:add", args=[app_label, model_name])
+    return reverse(f"wagtailsnippets_{app_label}_{model_name}:add")
```

Comment:
similar to compat.py -- drop these and replace their usage with direct `reverse()`

Reply:
Used direct reverse(), removed unnecessary code on `wagtail_localize/tests/utils.py`